core-libs/setup/ssr/error-handling/server-error-response/propagate-server-error-response.ts

```diff
@@ -5,12 +5,11 @@
  */
 
 import { InjectionToken } from '@angular/core';
-import { CxServerErrorResponse } from './cx-server-error-response';
 
 /**
  * Function that will propagate the server error response object
  * (created during server side rendering) to the CxCommonEngine.
  */
 export const PROPAGATE_SERVER_ERROR_RESPONSE = new InjectionToken<
-  (serverResponse: CxServerErrorResponse) => void
+  (serverResponse: unknown) => void
```

Comment:
We no longer talk about our custom ErrorResponse obejcts. Instead, we just forward raw Errors. Therefore the `ServerResponse` vocabulary should be removed from our Epic PR